presto-product-tests/src/main/java/io/prestosql/tests/hive/TestHiveTransactionalTable.java

```diff
@@ -407,26 +404,23 @@ public void testFilesForAbortedTransactionsIgnored()
             client.allocateTableWriteIds("default", tableName, Collections.singletonList(transaction)).get(0).getWriteId();
             client.abortTransaction(transaction);
 
-            String deltaA = warehouseDirectory + "/" + tableName + "/delta_0000001_0000001_0000/bucket_00000";
-            String deltaB = warehouseDirectory + "/" + tableName + "/delta_0000002_0000002_0000/bucket_00000";
-            String deltaC = warehouseDirectory + "/" + tableName + "/delta_0000003_0000003_0000/bucket_00000";
+            String deltaA = tableLocation + "/delta_0000001_0000001_0000";
+            String deltaB = tableLocation + "/delta_0000002_0000002_0000";
+            String deltaC = tableLocation + "/delta_0000003_0000003_0000";
 
             // Delete original `delta-B`, `delta-C`
-            hdfsClient.delete(deltaB);
-            hdfsClient.delete(deltaC);
-
-            ByteArrayOutputStream byteArrayOutputStream = new ByteArrayOutputStream();
-            hdfsClient.loadFile(deltaA, byteArrayOutputStream);
```

Comment:
why gone?

Reply:
Files are read as part of `hdfsCopyAll`

Reply:
here we were creating empty `delta_0000001_0000001_0000/bucket_00000` file. who does that now?

Reply:
No we were not. `delta_0000001_0000001_0000` were created on initial insert. And so it remains.
loadFile is 'read file from hdfs'. Yeah - naming could be better.